perception/navigator_vision/nodes/database_color.py

```diff
@@ -75,19 +83,23 @@ def _push(self, data):
             self._images.append(data)
             self._images = self._images[-self._history_length:]
 
-    def get_around_time(self, time, margin=.05):
+    def get_around_time(self, time, margin=.5, timeout=.5):
```

Comment:
To my knowledge rospy message filters don't work with TF since the /tf topic publishes raw transform messages. You'd have to do a message filter with the transform listener which i don't think you can do.
Correct me if I'm wrong